projects/backend/capi/articles.ts

```diff
@@ -61,7 +61,7 @@ const parseArticleResult = async (
     if (internalid == null)
         throw new Error(`internalid was undefined in ${path}!`)
 
-    const title = result && result.webTitle
+    const title = result && result.fields && result.fields.headline
```

Comment:
Maybe use webTitle if headline isn't present?

```suggestion
    const title = result && (result.fields && result.fields.headline || result.webTitle)
```